benchmarks/base.py

```diff
@@ -18,9 +19,14 @@ def sources_dir():
     return path
 
 
+@lru_cache(None)
+def get_data_for_file(path, file_size):
```

Comment:
I would leave a comment why we need `path`, mostly due to fact that it itches to get removed, and doing that will actualy break our benchmarks (we will have single file in all datasets, so the results will be in no way meaningful).

Reply:
also, isnt `lru_cache(None)` the same as `cache` ?

Reply:
> also, isnt lru_cache(None) the same as cache ?

This only available on 3.9, and I didn't know the version of the runner so went with the safe bet.

Reply:
Right, lets keep it that way